### ISSUE TITLE
Suggestions to run it against other datasets

### ISSUE THREAD

**Issue by Jurafsky21** [chiayewken/Span-ASTE#7]
Hi! I'm pretty new to deep learning and ASTE. 

Can you please suggest to me the necessary steps to run this against another dataset? 
Do I need to follow this data structure (https://github.com/xuuuluuu/SemEval-Triplet-data/blob/master/README.md#data-description) on my dataset by labeling it?
How can I modify the code on Colab for new datasets? thank you
Any other advice?

Thank you

**Comment by chiayewken** [comment]
Hi, yes you would need to annotate the data in the same format. In the folder "aste/data/triplet_data", you can create a folder called "new_data", and put train.txt, dev.txt and test.txt inside. Then, you can specify the new dataset for training by modifying line 11 in aste/main.sh to be "--names new_data, \" and line 12 to be "--seeds 0, \".

**Comment by chiayewken** [comment]
Hi, the data format that the training script needs is the same that is in Span-ASTE/aste/data/triplet_data/14lap/train.txt, which is like the sample below. The train, dev and test samples have the same format.

I charge it at night and skip taking the cord with me because of the good battery life .####I=O charge=O it=O at=O night=O and=O skip=O taking=O the=O cord=O with=O me=O because=O of=O the=O good=O battery=T-POS life=T-POS .=O####I=O charge=O it=O at=O night=O and=O skip=O taking=O the=O cord=O with=O me=O because=O of=O the=O good=S battery=O life=O .=O####[([16, 17], [15], 'POS')]

**Comment by chiayewken** [comment]
Hi, to make it more convenient to apply to new datasets, you can omit the tags component of the annotation, and include just the sentence and triplet information, such as the sample below. Each line in the train, dev and test set can have the same format.

I charge it at night and skip taking the cord with me because of the good battery life .#### #### ####[([16, 17], [15], 'POS')]

**Comment by Jurafsky21** [comment]
It works, thanks!

Last question: once I have my trained model in the model.tar, how can I
re-use it in a not labeled and unseen data?

Il giorno lun 10 gen 2022 alle ore 03:44 Chia Yew Ken <
***@***.***> ha scritto:
